src/main/java/frc/team2412/robot/commands/launcher/FullTargetCommand.java

```diff
@@ -58,6 +55,11 @@ public FullTargetCommand(
 	public void initialize() {
 		CommandScheduler.getInstance().schedule(yawAlignmentCommand);
 		intakeSubsystem.feederStop();
+
+		SPEAKER_POSE =
```

Comment:
For those curious about this change, this fix was required to resolve a crashing bug in simulation. Because the Alliance is reported as INVALID on sim startup, getting the alliance would crash in sim. This issue is only observed in simulation which is why it wasn't caught in its original PR.